sdk/resource/resource.go

```diff
@@ -109,6 +109,16 @@ func (r *Resource) String() string {
 	return r.attrs.Encoded(attribute.DefaultEncoder())
 }
 
+func (r *Resource) MarshalLog() interface{} {
+	return struct {
+		Attributes []attribute.KeyValue
+		SchemaURL  string
+	}{
+		Attributes: r.attrs.ToSlice(),
```

Comment:
How well does this render? Do we need to add marshaling instructions to the `attribute.KeyValue` type?

Reply:
You are right, the resources were marshaling as a list of objects with key, and value wasn't marsheled.  So something like
```
[{"Key": "Foo", "Value":{}}, {"Key":"Bar","Value":{}}]
```

I found a better fix would be to marshal the set as a `map[string]string` so we can get something like:
```
"Attributes":{"telemetry.sdk.language":"go",...,"service.name":"unknown_service:namedtracer"}
```